packages/cli/src/commands/publish/submit.ts

```diff
@@ -42,19 +43,38 @@ export const publishSubmitCommand = async ({
     release: releaseDetails,
     solana_mobile_dapp_publisher_portal: solanaMobileDappPublisherPortalDetails,
   } = await getConfigFile();
+
   const sign = ((buf: Buffer) =>
     nacl.sign(buf, signer.secretKey)) as SignWithPublisherKeypair;
 
-  await publishSubmit(
-    { connection, sign },
-    {
-      appMintAddress: appMintAddress ?? appDetails.address,
-      releaseMintAddress: releaseMintAddress ?? releaseDetails.address,
-      publisherDetails,
-      solanaMobileDappPublisherPortalDetails,
-      compliesWithSolanaDappStorePolicies,
-      requestorIsAuthorized,
-    },
-    dryRun
-  );
+  const pubAddr = publisherDetails.address;
+  const appAddr = appMintAddress ?? appDetails.address;
+  const releaseAddr = releaseMintAddress ?? releaseDetails.address;
+
+  try {
+    const results = await connection.getMultipleAccountsInfo([
```

Comment:
Added on-chain checking before publishing.

Reply:
Need to do this for `dapp-store publish update`, `dapp-store publish remove`, and `dapp-store publish support` as well.